extensions/windows-event-log/wel/WindowsEventLog.cpp

```diff
@@ -105,7 +105,6 @@ void WindowsEventLogMetadataImpl::renderMetadata() {
       default:
         event_type_index_ = 0;
     };
```

Comment:
not related to blank lines, but this `;` is not needed here and will cause a linter error at some point

Reply:
That is a different error of a different scope than the errors this PR meant to fix.